pkg/webui/console/components/contact/index.js

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+
+import Icon from '@ttn-lw/components/icon'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import style from './contact.styl'
+
+const Contact = props => {
+  if (!props.ids) {
+    return (
+      <i>
+        <Message content={sharedMessages.none} />
+      </i>
+    )
+  }
+  const isUser = 'user_ids' in props.ids
+  const icon = isUser ? 'user' : 'organization'
+  const id = isUser ? props.ids.user_ids.user_id : props.ids.organization_ids.organization_id
```

Comment:
```suggestion
  const id = getCollaboratorId({ids: props.ids})
```

from https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.17/pkg/webui/lib/selectors/id.js#L46-L48